<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="description" content="Abundance vs. occurrence data The typical output of a metabarcoding pipeline is an observation contingency table containing quantitative information of each type within each unit. Although researchers usually aim for even sequencing depths across study units, all unit are rarely represented by the same number of DNA sequences.
 An example abundance observation contingency table:
     Unit 1 Unit 2 Unit 3 &hellip; Unit n     Type 1 2332 34 44332 &hellip; 232   Type 2 0 23423 4433 &hellip; 3242   Type 3 32 3344 0 &hellip; 6855   &hellip;  &hellip; &hellip; &hellip; &hellip;   Type n 3543 0 133 &hellip; 1242   Total 33232 34554 24424 &hellip; 54567     The following R code creates a dummy observation contingency table:"><meta property="og:title" content="Abundance vs. occurrence data" />
<meta property="og:description" content="" />
<meta property="og:type" content="website" />
<meta property="og:url" content="http://example.org/docs/data-metadata/abundance-occurrence/" />

<title>Abundance vs. occurrence data | Metabarcoding diversity</title>
<link rel="icon" href="/favicon.png" type="image/x-icon">
<link rel="stylesheet" href="/book.min.2d9191914ee9031d630c7f7e96d8662ea6b18e228cd0bce75cea28e489496f28.css" integrity="sha256-LZGRkU7pAx1jDH9&#43;lthmLqaxjiKM0LznXOoo5IlJbyg=">
<script defer src="/en.search.min.daf152d79f4bf7bf9d32b850b6e313a5b4f6f2f8e2b50628235ca64f9a333045.js" integrity="sha256-2vFS159L97&#43;dMrhQtuMTpbT28vjitQYoI1ymT5ozMEU="></script>
<link rel="alternate" type="application/rss+xml" href="http://example.org/docs/data-metadata/abundance-occurrence/index.xml" title="Metabarcoding diversity" />
<!--
Made with Book Theme
https://github.com/alex-shpak/hugo-book
-->

  
</head>

<body>
  <input type="checkbox" class="hidden" id="menu-control" />
  <main class="container flex">
    <aside class="book-menu">
      
  <nav>










  

  
  





 
  
    




  
  <ul>
    
      
        

  <li class="book-section-flat" >
    

  
  <a href="/docs/dna-metabarcoding/" class="">DNA metabarcoding</a>
  


    




  
  <ul>
    
  </ul>
  



  </li>


      
    
      
        

  <li class="book-section-flat" >
    

  
  <a href="/docs/data-metadata/" class="">Data and metadata</a>
  


    




  
  <ul>
    
      
        

  <li>
    

  
  <a href="/docs/data-metadata/abundance-occurrence/" class="active">Abundance vs. occurrence data</a>
  


    




  
  <ul>
    
  </ul>
  



  </li>


      
    
      
        

  <li>
    

  
  <a href="/docs/data-metadata/collapsed/" class="collapsed ">Collapsed</a>
  


    






  </li>


      
    
  </ul>
  



  </li>


      
    
      
        

  <li class="book-section-flat" >
    

  
  <a href="/docs/diversity-metrics/" class="">Diversity metrics</a>
  


    




  
  <ul>
    
      
        

  <li>
    

  
  <a href="/docs/diversity-metrics/diversity-indices/" class="">Diversity indices</a>
  


    




  
  <ul>
    
  </ul>
  



  </li>


      
    
      
        

  <li>
    

  
  <a href="/docs/diversity-metrics/phylodiversity-indices/" class="">Phylodiversity indices</a>
  


    




  
  <ul>
    
  </ul>
  



  </li>


      
    
      
        

  <li>
    

  
  <a href="/docs/diversity-metrics/hill-numbers/" class="collapsed ">Hill numbers</a>
  


    






  </li>


      
    
      
        

  <li>
    

  
  <a href="/docs/diversity-metrics/type-correlations/" class="">Type correlations</a>
  


    




  
  <ul>
    
  </ul>
  



  </li>


      
    
      
        

  <li>
    

  
  <a href="/docs/diversity-metrics/diversity-decomposition/" class="">Diversity decomposition</a>
  


    




  
  <ul>
    
  </ul>
  



  </li>


      
    
  </ul>
  



  </li>


      
    
      
        

  <li class="book-section-flat" >
    

  
  <a href="/docs/diversity-partitioning/" class="">Diversity partitioning</a>
  


    




  
  <ul>
    
      
        

  <li>
    

  
  <a href="/docs/diversity-partitioning/phylodiversity-partitioning/" class="">Phylogenetic diversity partitioning</a>
  


    




  
  <ul>
    
  </ul>
  



  </li>


      
    
  </ul>
  



  </li>


      
    
      
        

  <li class="book-section-flat" >
    

  
  <a href="/docs/dissimilarity-measurement/" class="">Dissimilarity measurement</a>
  


    




  
  <ul>
    
  </ul>
  



  </li>


      
    
      
        

  <li class="book-section-flat" >
    

  
  <a href="/docs/case-studies/" class="">Case studies</a>
  


    




  
  <ul>
    
      
        

  <li>
    

  
  <a href="/docs/case-studies/ecosystem-diversity/" class="">Ecosystem diversity</a>
  


    




  
  <ul>
    
  </ul>
  



  </li>


      
    
      
        

  <li>
    

  
  <a href="/docs/case-studies/dietary-diversity/" class="">Dietary diversity</a>
  


    




  
  <ul>
    
  </ul>
  



  </li>


      
    
      
        

  <li>
    

  
  <a href="/docs/case-studies/microbiome-diversity/" class="">Microbiome diversity</a>
  


    




  
  <ul>
    
  </ul>
  



  </li>


      
    
  </ul>
  



  </li>


      
    
      
        

  <li class="book-section-flat" >
    

  
  <a href="/docs/authors/" class="">Authors</a>
  


    




  
  <ul>
    
  </ul>
  



  </li>


      
    
      
        

  <li class="book-section-flat" >
    

  
  <a href="/docs/datasets/" class="">Data sets</a>
  


    




  
  <ul>
    
  </ul>
  



  </li>


      
    
      
        

  <li class="book-section-flat" >
    

  
  <a href="/docs/glossary/" class="">Glossary</a>
  


    




  
  <ul>
    
  </ul>
  



  </li>


      
    
  </ul>
  



  












  
<ul>
  
  <li>
    <a href="/tags/" >
        Tags
      </a>
  </li>
  
  <li>
    <a href="/posts/" >
        Blog
      </a>
  </li>
  
</ul>






</nav>




  <script>(function(){var menu=document.querySelector("aside.book-menu nav");addEventListener("beforeunload",function(event){localStorage.setItem("menu.scrollTop",menu.scrollTop);});menu.scrollTop=localStorage.getItem("menu.scrollTop");})();</script>


 
    </aside>

    <div class="book-page">
      <header class="book-header">
        
  <div class="flex align-center justify-between">
  <label for="menu-control">
    <img src="/svg/menu.svg" class="book-icon" alt="Menu" />
  </label>

  <strong>Abundance vs. occurrence data</strong>

  <label for="toc-control">
    <img src="/svg/toc.svg" class="book-icon" alt="Table of Contents" />
  </label>
</div>


  
    <input type="checkbox" class="hidden" id="toc-control" />
    <aside class="hidden clearfix">
      
  <nav id="TableOfContents">
  <ul>
    <li><a href="#abundances-to-occurrences">Abundances to occurrences</a></li>
    <li><a href="#normalising-absolute-matrices">Normalising absolute matrices</a>
      <ul>
        <li><a href="#total-sum-scaling">Total sum scaling</a></li>
        <li><a href="#other-methods">Other methods</a></li>
      </ul>
    </li>
  </ul>
</nav>


    </aside>
  
 
      </header>

      <div style="border-bottom:1.2px solid #cecece; margin-bottom:10px; color: #999999;" align="center"><a href="/">Diversity analysis of DNA metabarcoding data</a></div>


<div class="book-search">
  <input type="text" id="book-search-input" placeholder="Search" aria-label="Search" maxlength="64" data-hotkeys="s/" />
  <div class="book-search-spinner spinner hidden"></div>
  <ul id="book-search-results"></ul>
</div>



      <div id="breadcrumbs" style="border-left:3px solid #78b3ba; padding:5px; margin-bottom:20px; color:#78b3ba">
    <a href="/">Home</a>
    
        
    
        
            > <a href="/docs">Docs</a>
        
    
        
            > <a href="/data-metadata">Data metadata</a>
        
    
        
            > <a href="/abundance-occurrence">Abundance occurrence</a>
        
    
        
    
</div>

      
<div class="tags" style="padding-left:15px">

  <a class="tag-link" href="/tags/abundance">abundance</a>

  <a class="tag-link" href="/tags/occurrence">occurrence</a>

  <a class="tag-link" href="/tags/transformation">transformation</a>

  <a class="tag-link" href="/tags/normalisation">normalisation</a>

</div>


      
  <article class="markdown"><h1 id="abundance-vs-occurrence-data">Abundance vs. occurrence data</h1>
<p>The typical output of a metabarcoding pipeline is an observation contingency table containing quantitative information of each type within each unit. Although researchers usually aim for even sequencing depths across study units, all unit are rarely represented by the same number of DNA sequences.</p>
<blockquote>
<p>An example abundance observation contingency table:</p>
</blockquote>
<table>
<thead>
<tr>
<th></th>
<th>Unit 1</th>
<th>Unit 2</th>
<th>Unit 3</th>
<th>&hellip;</th>
<th>Unit n</th>
</tr>
</thead>
<tbody>
<tr>
<td>Type 1</td>
<td>2332</td>
<td>34</td>
<td>44332</td>
<td>&hellip;</td>
<td>232</td>
</tr>
<tr>
<td>Type 2</td>
<td>0</td>
<td>23423</td>
<td>4433</td>
<td>&hellip;</td>
<td>3242</td>
</tr>
<tr>
<td>Type 3</td>
<td>32</td>
<td>3344</td>
<td>0</td>
<td>&hellip;</td>
<td>6855</td>
</tr>
<tr>
<td>&hellip;</td>
<td></td>
<td>&hellip;</td>
<td>&hellip;</td>
<td>&hellip;</td>
<td>&hellip;</td>
</tr>
<tr>
<td>Type n</td>
<td>3543</td>
<td>0</td>
<td>133</td>
<td>&hellip;</td>
<td>1242</td>
</tr>
<tr>
<td><strong>Total</strong></td>
<td><strong>33232</strong></td>
<td><strong>34554</strong></td>
<td><strong>24424</strong></td>
<td><strong>&hellip;</strong></td>
<td><strong>54567</strong></td>
</tr>
</tbody>
</table>
<blockquote>
<p>The following R code creates a dummy observation contingency table:</p>
</blockquote>
<div class="highlight"><pre style="color:#f8f8f2;background-color:#272822;-moz-tab-size:4;-o-tab-size:4;tab-size:4"><code class="language-R" data-lang="R">abundance.matrix <span style="color:#f92672">&lt;-</span> <span style="color:#a6e22e">cbind</span>(Predator1<span style="color:#f92672">=</span><span style="color:#a6e22e">c</span>(<span style="color:#ae81ff">10</span>,<span style="color:#ae81ff">1</span>,<span style="color:#ae81ff">20</span>,<span style="color:#ae81ff">30</span>),Predator2<span style="color:#f92672">=</span><span style="color:#a6e22e">c</span>(<span style="color:#ae81ff">10</span>,<span style="color:#ae81ff">0</span>,<span style="color:#ae81ff">10</span>,<span style="color:#ae81ff">0</span>),Predator3<span style="color:#f92672">=</span><span style="color:#a6e22e">c</span>(<span style="color:#ae81ff">90</span>,<span style="color:#ae81ff">45</span>,<span style="color:#ae81ff">29</span>,<span style="color:#ae81ff">5</span>))
<span style="color:#a6e22e">rownames</span>(abundance.matrix) <span style="color:#f92672">&lt;-</span> <span style="color:#a6e22e">c</span>(<span style="color:#e6db74">&#34;Prey1&#34;</span>,<span style="color:#e6db74">&#34;Prey2&#34;</span>,<span style="color:#e6db74">&#34;Prey3&#34;</span>,<span style="color:#e6db74">&#34;Prey4&#34;</span>)
</code></pre></div><h2 id="abundances-to-occurrences">Abundances to occurrences</h2>
<p>This original abundance matrix can be transformed into an occurrence matrix, by converting all non-zero values into ones. This second matrix does not consider DNA read counts, but simply records either detection (1) or non-detection (0) of each prey in each predator.</p>
<blockquote>
<p>An example occurrence observation contingency table:</p>
</blockquote>
<table>
<thead>
<tr>
<th></th>
<th>Unit 1</th>
<th>Unit 2</th>
<th>Unit 3</th>
<th>&hellip;</th>
<th>Unit n</th>
</tr>
</thead>
<tbody>
<tr>
<td>Type 1</td>
<td>1</td>
<td>1</td>
<td>1</td>
<td>&hellip;</td>
<td>1</td>
</tr>
<tr>
<td>Type 2</td>
<td>0</td>
<td>1</td>
<td>1</td>
<td>&hellip;</td>
<td>1</td>
</tr>
<tr>
<td>Type 3</td>
<td>1</td>
<td>1</td>
<td>0</td>
<td>&hellip;</td>
<td>1</td>
</tr>
<tr>
<td>&hellip;</td>
<td></td>
<td>&hellip;</td>
<td>&hellip;</td>
<td>&hellip;</td>
<td>&hellip;</td>
</tr>
<tr>
<td>Type n</td>
<td>1</td>
<td>0</td>
<td>1</td>
<td>&hellip;</td>
<td>1</td>
</tr>
<tr>
<td><strong>Total</strong></td>
<td><strong>35</strong></td>
<td><strong>30</strong></td>
<td><strong>123</strong></td>
<td><strong>&hellip;</strong></td>
<td><strong>58</strong></td>
</tr>
</tbody>
</table>
<blockquote>
<p>The following R code transforms the abundance matrix into an occurrence matrix:</p>
</blockquote>
<div class="highlight"><pre style="color:#f8f8f2;background-color:#272822;-moz-tab-size:4;-o-tab-size:4;tab-size:4"><code class="language-R" data-lang="R">occurrence.matrix <span style="color:#f92672">&lt;-</span> abundance.matrix
occurrence.matrix[occurrence.matrix <span style="color:#f92672">!=</span> <span style="color:#ae81ff">0</span>] <span style="color:#f92672">&lt;-</span> <span style="color:#ae81ff">1</span>
</code></pre></div><h2 id="normalising-absolute-matrices">Normalising absolute matrices</h2>
<h3 id="total-sum-scaling">Total sum scaling</h3>
<h3 id="other-methods">Other methods</h3>
</article>
 
      <div style="background: #fcfcfc; border-top:1px solid #f4f4f4; padding:5px; margin-top:20px; color:#666666" align="center">2020 CC-by-sa. Antton Alberdi & Ostaizka Aizpurua</div>


      <footer class="book-footer">
        
  <div class="flex justify-between">





</div>

 
        
      </footer>

      
  
  <div class="book-comments">

</div>
  
 

      <label for="menu-control" class="hidden book-menu-overlay"></label>
    </div>

    
    <aside class="book-toc">
      
  <nav id="TableOfContents">
  <ul>
    <li><a href="#abundances-to-occurrences">Abundances to occurrences</a></li>
    <li><a href="#normalising-absolute-matrices">Normalising absolute matrices</a>
      <ul>
        <li><a href="#total-sum-scaling">Total sum scaling</a></li>
        <li><a href="#other-methods">Other methods</a></li>
      </ul>
    </li>
  </ul>
</nav>

 
    </aside>
    
  </main>

  
</body>

</html>












